src/libraries/System.Numerics.Tensors/tests/TensorPrimitives.Generic.cs

```diff
@@ -365,7 +365,7 @@ public static IEnumerable<object[]> SpanDestinationFunctionsToTest()
             yield return new object[] { new SpanDestinationDelegate(TensorPrimitives.Acosh), new Func<T, T>(T.Acosh) };
             yield return new object[] { new SpanDestinationDelegate(TensorPrimitives.AcosPi), new Func<T, T>(T.AcosPi) };
             yield return new object[] { new SpanDestinationDelegate(TensorPrimitives.Acos), new Func<T, T>(T.Acos) };
-            yield return new object[] { new SpanDestinationDelegate(TensorPrimitives.Asinh), new Func<T, T>(T.Asinh) };
+            yield return new object[] { new SpanDestinationDelegate(TensorPrimitives.Asinh), new Func<T, T>(T.Asinh), T.CreateTruncating(0.001) };
```

Comment:
This is quite a lot of variance being allowed, we're basically saying we can't get within 3 digits of accuracy for some inputs.

Reply:
What do you suggest?

(Note that the default being used everywhere right now is 0.0001.)